db/migrate/20220524004300_create_table_retractions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTableRetractions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :table_retractions do |t|
```

Comment:
the table name should be `retractions` not `table_retractions`

Reply:
Did db:rollback to fix table name and did migration after it.